lib/consoletest.pm

```diff
@@ -16,5 +16,11 @@ sub post_run_hook {
     $self->clear_and_verify_console;
 }
 
+sub post_fail_hook {
+    # Export logs after failure
+    script_run "journalctl --no-pager -b 0 > /tmp/full_journal.log";
```

Comment:
and please check the modules which include this module as base module where they would overwrite this post_fail_hook